bedrock/firefox/templates/firefox/new/desktop/firefox-new-refresh-all.html

```diff
@@ -0,0 +1,267 @@
+      <div class="c-block-container">
+        <div class="c-block-body l-v-center">
+          <h3 class="mzp-u-title-sm">{{ ftl('firefox-new-refresh-your-stuff') }}</h3>
+          <p>{{ ftl('firefox-new-refresh-get-firefox-mibile') }}</p>
+          <div class="mobile-download-buttons-wrapper">
+            {% block mobile_secondary_cta %}
+              <ul class="mobile-download-buttons">
+                <li class="android">
+                  {{ google_play_button() }}
+                </li>
+                <li class="ios">
+                  {{ apple_app_store_button(href=ios_url) }}
+                </li>
+              </ul>
+            {% endblock %}
+          </div>
+        </div>
+        <div class="c-block-media l-v-end l-h-end l-media-constrain-on-sm">
+        <img alt="Desktop, laptop, and phone. " class="c-block-media-img" src="{{ static('img/firefox/new/desktop/firefox-new-refresh/screens.svg') }}" width="350">
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <div class="t-custom">
+  <section class="mzp-l-content mzp-t-content-md">
+    <h2 class="mzp-c-section-heading">{{ ftl('firefox-new-refresh-really-make-it') }}</h2>
+
+    <div class="c-screen">
+      <div class="c-screenshot">
+        <img alt="Firefox in dark mode." src="{{ static('img/firefox/new/desktop/firefox-new-refresh/dark.svg') }}" width="1000">
+      </div>
+    </div>
+
+    <ul class="mzp-l-columns mzp-t-columns-two">
+      <li>
+        <h3>{{ ftl('firefox-new-refresh-work-smarter') }}</h3>
+        <p>{{ ftl('firefox-new-refresh-explore-the-possibilities', extensions_url='https://addons.mozilla.org/firefox/extensions/') }}</p>
+      </li>
+
+      <li>
+        <h3>{{ ftl('firefox-new-refresh-bye-boring') }}</h3>
+        <p>{{ ftl('firefox-new-refresh-the-internet-can', theme_url='https://addons.mozilla.org/firefox/themes/') }}</p>
+      </li>
+    </ul>
+  </section>
+</div>
+
+  <section class="mzp-l-content t-free">
+  <div class="mzp-c-emphasis-box js-animate">
+    <div class="c-free">
+      <div class="c-free-img">
+        <img alt="Firefox on a desktop." src="{{ static('img/firefox/new/desktop/firefox-new-refresh/free.svg') }}" width="300">
+      </div>
+      <div class="c-free-body">
+        <h2>{{ ftl('firefox-new-refresh-billionaire-free') }}</h2>
+        <p>{{ ftl('firefox-new-refresh-firefox-was-created') }}</p>
+      </div>
+    </div>
+  </div>
+</section>
+
+  <section class="mzp-c-section-heading">
+    {% block discover_cta %}
+      {{ default_browser_checkbox(id='default-browser-checkbox-discover') }}
+      {{ download_firefox_thanks(dom_id='download-discover', locale_in_transition=True, download_location='discover cta') }}
+    {% endblock %}
+  </section>
+
+  <section class="c-support">
+    {% set questions_attrs = 'href="https://support.mozilla.org/products/firefox%s&utm_content=mozilla-support" rel="external noopener" data-cta-text="Mozilla support"'|safe|format(referrals) %}
```

Comment:
I'm not entirely sure here, but I think they _do_ translate the "Mozilla support" term.